vrecord.rb

```diff
@@ -40,6 +40,20 @@ def install
     man1.install "vtest.1"
   end
 
+  on_linux do
+    def caveats
+    <<~EOS
+      ** IMPORTANT FOR LINUX INSTALL **
+      Additional install steps are necessary for a fully functioning Vrecord
+      install on Linux. This includes using the standard package manager to
+      install gnuplot, xmlstarlet, mkvtoolnix and mediaconch. Additionally,
```

Comment:
On my end, `gnuplot`, `xmlstarlet` and` mediaconch` are working as expected on Debian with Brew, only `mkvtoolnix` makes little troubles here (I presume, Moritz Bunkus knows that).